packages/story-editor/src/components/form/color/color.js

```diff
@@ -131,6 +131,11 @@ const Color = forwardRef(function Color(
       ? TOOLTIP_PLACEMENT.BOTTOM
       : TOOLTIP_PLACEMENT.BOTTOM_START;
 
+  const Space = styled.div`
```

Comment:
This `styled.div` component should be defined outside of the `Color` component, otherwise a warning about 'dynamic' components shows in the browser console.

```
The component color__Space with the id of "color__Space-sc-c7i660-4" has been created dynamically.
You may see this warning because you've called styled inside another component.
To resolve this only create new StyledComponents outside of any render method and function component.
```
😶

Reply:
Ooh great feedback, thank you! I'll do just that!